app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1390,6 +1390,31 @@ body.dashboard-faq dl dt {
   border-bottom: none;
 }
 
+a:focus {
```

Comment:
I have not changed one line of CSS in Codebar planner and I have never used foundation!

Is it _possible_ to fit it into `assets/stylesheets/partials/_links.scss ` and while you're at it. change the code to use SCSS?

Something like:

```scss
a {
    &:hover {
       etc etc 
    }
   &:focus {
       your stuff etc etc
    }
} 
```